change/react-native-platform-override-2020-05-27-09-45-07-checkout-master.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
Ideally, deletion of this file must not be part of your change.

Reply:
Tried to undo a previous run of locally, should resolve itself when I merge for the copyright header.

Reply:
It didn't remove yet, so I rebased the removal out of the first commit.